openassessment/workflow_batch_update_api.py

```diff
@@ -0,0 +1,640 @@
+"""
+Provides functionality to batch update ORA workflows for different scopes
+"""
+
+import logging
+import time
+import datetime
+from django.utils import timezone
+from celery import shared_task
+
+from opaque_keys.edx.keys import UsageKey
+from openassessment.runtime_imports.functions import modulestore
+from openassessment.assessment.models import PeerWorkflow
+from openassessment.workflow import api
+
+logger = logging.getLogger(__name__)
+
+
+@shared_task(bind=True,
+             acks_late=True,
+             autoretry_for=(Exception,),
+             max_retries=3,
+             retry_backoff=True,
+             retry_backoff_max=500,
+             retry_jitter=True)
+def update_workflows_for_all_blocked_submissions_task(self):  # pylint: disable=unused-argument
+    """
+    Async task wrapper
+    """
+    return update_workflows_for_all_blocked_submissions()
```

Comment:
This and the other tasks should be in tasks.py

Reply:
They should, they were but with nested/chained async call it is not possible (python imports cycle)